www/badges/buildbot_badges/templates/flat-square.svg.j2

```diff
@@ -5,6 +5,6 @@
   <rect width="{{ left.width + right.width }}" height="20" fill-opacity=".1"/>
   <g fill="#fff" text-anchor="middle" font-family="{{config.font_face}},Verdana,Geneva,sans-serif" font-size="{{config.font_size}}">
     <text x="{{ left.width/2+1 }}" y="14">{{ left.text }}</text>
-    <text x="{{ left.width+right.width/2-1 }}" y="14">{{ right.text }}</text>
+    <text x="{{ right.width/2+left.width-1 }}" y="14">{{ right.text }}</text>
```

Comment:
I wonder if there could be a bit more descriptive commit message justifying the change: if I read it right, the expression looks the same, so a bit more reasoning behind the change would be more than welcome.

Reply:
The expression is actually quite different:  You have two parts, `left` and `right`.  We want the text for `right` to be centred within `right`.  So the X position of right's font is `right.width/2 - 1`, offset from zero by `left.width`.  The previous expression took with width of the final image, and plonked the text down in the middle of it :)

I can take a look at redoing the commits with the above explanation.

Reply:
So this is a JavaScript thingie?..  Assuming it is, resolving this conversation.